puppetserver-ca.gemspec

```diff
@@ -13,11 +13,11 @@ Gem::Specification.new do |spec|
   spec.summary       = %q{A simple CLI tool for interacting with Puppet Server's Certificate Authority}
   spec.homepage      = "https://github.com/puppetlabs/puppetserver-ca-cli/"
 
-  spec.files         = `git ls-files -z`.split("\x0").reject do |f|
-    f.match(%r{^(test|spec|features)/})
+  spec.files         = `git ls-files -z`.split("\x0").select do |f|
+    f.start_with?('LICENSE', 'README.md', 'lib/')
```

Comment:
I think the `exe` directory is missing here :thinking:.

Reply:
I used `gem build` and they show up. There was no need to list it twice (other is via executables)